bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookies/InfoCommand.java

```diff
@@ -80,14 +82,27 @@ public boolean apply(ServerConfiguration conf, CliFlags cmdFlags) {
 
             System.out.println("Free disk space info:");
             long totalFree = 0, total = 0;
-            for (Map.Entry<BookieSocketAddress, BookieInfo> e : map.entrySet()) {
-                BookieInfo bInfo = e.getValue();
-                BookieSocketAddress bookieId = e.getKey();
-                System.out.println(CommandHelpers.getBookieSocketAddrStringRepresentation(bookieId)
-                    + ":\tFree: " + bInfo.getFreeDiskSpace() + getReadable(bInfo.getFreeDiskSpace())
-                    + "\tTotal: " + bInfo.getTotalDiskSpace() + getReadable(bInfo.getTotalDiskSpace()));
-                totalFree += bInfo.getFreeDiskSpace();
-                total += bInfo.getTotalDiskSpace();
+
+            Map<String, BookieInfoReader.BookieInfo> dedupedMap = map.entrySet()
+                    .stream()
+                    .map(e -> {
+                        BookieInfoReader.BookieInfo bInfo = e.getValue();
+                        BookieSocketAddress bookieId = e.getKey();
+                        System.out.println(CommandHelpers.getBookieSocketAddrStringRepresentation(bookieId)
```

Comment:
It is not a good practice to output data in 'map'

Maybe if you do not use streams API code will look better

Reply:
@eolivelli Got, done.